modules/widgets/twitter-timeline.php

```diff
@@ -362,11 +385,11 @@ class="widefat"
 		</p>
 
 		<p>
-			<label for="<?php echo $this->get_field_id( 'height' ); ?>">
+			<label for="<?php echo $this->get_field_id( 'height' ); ?>" <?= $has_tweet_limit ? 'class="jp-twitter-disabled"' : '' ?>>
```

Comment:
You'll want to avoid using shorthand PHP tags, as per the WordPress coding standards:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#no-shorthand-php-tags